package.json

```diff
@@ -220,6 +220,12 @@
                     "description": "Whether to add the local npm bin directory to the PATH for purs IDE server and build command.",
                     "scope": "resource"
                 },
+                "purescript.fullBuildOnSave": {
```

Comment:
This is not relevant to the topic of this PR

Reply:
Oops, that's cruft from my experimenting, deleted!